markdown/reference/sql/DROP-CONVERSION.html.md.erb

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
This makes three times I've changed this file. I don't know what's going on, but it keeps reverting to the file I used as a template. I'm going to try changing it outside the repo, then copying it to overwrite the one in the repo and see if that works.